apps/remix-ide/src/remixAppManager.js

```diff
@@ -14,7 +14,7 @@ const requiredModules = [ // services + layout views + system views
 const dependentModules = ['git', 'hardhat'] // module which shouldn't be manually activated (e.g git is activated by remixd)
 
 export function isNative (name) {
-  const nativePlugins = ['vyper', 'workshops', 'debugger', 'remixd', 'menuicons']
+  const nativePlugins = ['vyper', 'workshops', 'debugger', 'remixd', 'menuicons', 'solidity']
```

Comment:
not sure if `solidity-logic` should be added instead of `solidity`

Reply:
this `isNative` is used in the `canActivate` function, it allows solidity to activate solidity-logic

Reply:
ok